### DISCUSSION TITLE
280Ah Cell

### DISCUSSION THREAD

**Discussion by bisrael8191** [pybamm-team/PyBaMM#2243] [Q&A] [answered]
I'm trying to configure the Prada2013 model to more closely match a 280Ah, 1C cell instead of the default 1.1Ah. I've played around with changing a bunch of different parameters, but a naive approach (below) of just changing the current and voltage parameters causes the solver to hang forever with no output or errors. Does anyone know the correct way to scale the cell capacity such that the voltages and current are realistic during charge/discharge cycles? 

The code below is using the latest release, v22.7.

```python3
import pybamm


if __name__ == "__main__":
    options = {"calculate discharge energy": "true", "cell geometry": "arbitrary"}
    model = pybamm.lithium_ion.SPMe(options=options)
    param = pybamm.ParameterValues(chemistry=pybamm.parameter_sets.Prada2013)
    param["Nominal cell capacity [A.h]"] = 280.0
    param["Current function [A]"] = 280.0
    param["Typical current [A]"] = 280.0
    param["Upper voltage cut-off [V]"] = 3.65
    param["Lower voltage cut-off [V]"] = 2.5
    param.process_model(model)

    exp = pybamm.Experiment([
        "Charge at 1C until 3.65V",
        "Rest for 60 seconds",
    ])
    initial_soc = 0.9
    sim = pybamm.Simulation(
        model=model,
        experiment=exp,
        parameter_values=param,
        solver=pybamm.CasadiSolver()
    )
    sol = sim.solve(initial_soc=initial_soc)

    sim.plot()
```

**Comment by brosaplanella** [comment, marked as answer]
The nominal cell capacity is only used to calculate the C-rate. To change the actual capacity of the battery you need to change the electrode plate area, which is defined as the product between `Electrode width [m]` and `Electrode height [m]`.
